src/woo_publications/config/validators.py

```diff
@@ -0,0 +1,39 @@
+"""
+Copied/vendored validators from vng_api_common.validators which can't be imported
+on Python 3.12+.
+"""
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+
+RSIN_LENGTH = 9
+
+validate_digits = RegexValidator(
+    regex="^[0-9]+$", message="Waarde moet numeriek zijn.", code="only-digits"
+)
+
+
+def validate_rsin(value):
+    """
+    Validates that a string value is a valid RSIN number by applying the
+    '11-proef' checking.
+
+    :param value: String object representing a presumably good RSIN number.
+    """
+    # Initial sanity checks.
+    validate_digits(value)
+    if len(value) != RSIN_LENGTH:
+        raise ValidationError(
+            "RSIN moet %s tekens lang zijn." % RSIN_LENGTH, code="invalid-length"
```

Comment:
Any reason why you added multiple dutch error messages which aren't translatable?

Reply:
blind copy-paste from vng-api-common! I'm hopeful that when OAF/commonground-api-common is patched with 3.12 compatibility we can then just import the existing validator

Reply:
It's probably smart to add a TODO comment then so we won't forget about it.